web/templates/pages/project_edit.html

```diff
@@ -6,11 +6,17 @@
 {% endblock %}
 
 {% block content %}
-	<h1 class="pull-left">{% if editing %}Edit{% else %}Add {% endif %} your project</h1>
-	<form method="POST">{% csrf_token %}
-		{{ projectform.as_p }}
-		<button type="submit" class="save btn btn-default">Save</button>
-	</form>
+
+	{% if project and editing %}
```

Comment:
Seems wrong but try to add  {% if not project and editing %} display the warning also